src/protocol/id/utxo.md

```diff
@@ -9,10 +9,10 @@ Is represented as an _outpoint_: a pair of [transaction ID](./transaction.md) as
 The ID of a message is computed as the [hash](../cryptographic_primitives.md#hashing) of:
 
 1. the sender address as `byte[32]`,
-1. the recipient address as `byte[32]`,
-1. the [Message nonce](#message-nonce) as `byte[32]`,
-1. the amount being sent with the message as `uint64`,
-1. the message data as `byte[]`
+2. the recipient address as `byte[32]`,
+3. the [Message nonce](#message-nonce) as `byte[32]`,
+4. the amount being sent with the message as `uint64`,
+5. the message data as `byte[]`
```

Comment:
Better to have `1.`. Markdown will increment it automatically

Reply:
I'll revert this change. I see that this is better for diffs when a new item gets added in the middle of the list.